README.md

```diff
@@ -3,10 +3,30 @@ Welcome to the Kokkos Documentation repository.  This is the source for https://
 
 ## Requirements to build html page locally
 
+The documentation requires Python 3.12 to build. Usually this can be installed with a system package manager, but if your system does not support python 3.12 you can install it easily with [pyenv](https://github.com/pyenv/pyenv).
 This is needed just for local render of documentation, so it can be checked before push.
 Requirements are in `build_requirements.txt`
 Could be installed with: `pip install -r build_requirements.txt`
 
+We recommend using a virtual environment, e.g. if your system python is >= 3.12:
+
+```sh
+python -m venv venv
```

Comment:
Is it recommended practice to call it "venv" as a default name?

Reply:
https://docs.python.org/3/library/venv.html

It says 
>  Contained in a directory, conventionally named .venv or venv in the project directory, or under a container directory for lots of virtual environments, such as ~/.virtualenvs.

So I just picked one but you can name it anything you like